exercises/resistor-color-duo/description.md

```diff
@@ -2,19 +2,9 @@ If you want to build something using a Raspberry Pi, you'll probably use _resist
 
 * Each resistor has a resistance value.
 * Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
-To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band acts as a digit of a number. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
+To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band has a position and a numeric value. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to the number 15.
```

Comment:
```suggestion
To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band has a position and a numeric value. For example, if they printed a brown band (value 1) followed by a green band (value 5), it would translate to a value of 15.
```

Perhaps?

Reply:
> "The original story has been written by the product team, to make it an exemplar for future descriptions, so let's keep that as is. "

We discussed `number` vs `value` in the original story discussion. Do you mind if we just leave it?

Reply:
Sure. I do think it _detracts_ from the clarity, but no strong opinion on it 😄